src/components/composer/grid_composer/grid_composer.ts

```diff
@@ -91,15 +90,18 @@ export class GridComposer extends Component<Props, SpreadsheetChildEnv> {
   get containerStyle(): string {
     const isFormula = this.env.model.getters.getCurrentContent().startsWith("=");
     const cell = this.env.model.getters.getActiveCell();
-    let style: Style = {};
-    if (cell) {
-      const cellPosition = this.env.model.getters.getCellPosition(cell.id);
-      style = this.env.model.getters.getCellComputedStyle(
-        cellPosition.sheetId,
-        cellPosition.col,
-        cellPosition.row
-      );
-    }
+    const position = this.env.model.getters.getPosition();
```

Comment:
In another commit, I would rename it to `getActivePosition` to be coherent with `getActiveSheetId`, `getActiveCell`, ...